Sources/NestKit/GitHub/ExcludedVersion.swift

```diff
@@ -0,0 +1,19 @@
+/// {owner}/{repo} and (optional) version pairs to exclude
+public struct ExcludedVersion: Equatable, Sendable {
```

Comment:
I doubt the struct really represents a version.
It's a bit unnatural for me that `***Version` has `version` internally, and the `version` property is optional in this case.

`target` seems better.

Reply:
Thanks!
I've renamed `version` to `target` here: https://github.com/mtj0928/nest/pull/33/commits/0333ade7f842d7c9a764173ff475c0f2e9503ae8

Reply:
Oh sorry, my comment Is a bit ambiguous.
I meant the type name like this.
```swift
public struct ExcludedTarget: Equatable, Sendable { 
    public let reference: String
    public let version: String?
}
```